template/frameworks/vuetify/layouts/error.vue

```diff
@@ -0,0 +1,46 @@
+<template>
+  <v-app dark>
+    <h1 v-if="error.statusCode === 404">
+      {{ this.pageNotFound }}
+    </h1>
+    <h1 v-else>
+      {{ this.otherError }}
+    </h1>
+    <NuxtLink to="/">
+      Home page
+    </NuxtLink>
+  </v-app>
+</template>
+
+<script>
+import pkg from '../package'
+export default {
+  layout: 'empty',
+  props: {
+    error: {
+      type: Object,
+      default: null
+    }
+  },
+  head () {
+    const title = this.error.statusCode === 404
+      ? pkg.name + ' | ' + this.pageNotFound
```

Comment:
Maybe we can use `titleTemplate` and avoid importing `pkg`

Reply:
Thank you !
I fix this problem.